pkg/cmd/plan.go

```diff
@@ -0,0 +1,276 @@
+package cmd
+
+import (
+	"errors"
+	"fmt"
+	"net/url"
+	"os"
+	"path/filepath"
+	"text/template"
+
+	"github.com/testground/testground/pkg/api"
+	"github.com/testground/testground/pkg/config"
+
+	ttmpl "github.com/testground/plan-templates/templates"
+
+	"github.com/BurntSushi/toml"
+	"github.com/go-git/go-git/v5"
+	gitcfg "github.com/go-git/go-git/v5/config"
+	"github.com/mattn/go-zglob"
+	"github.com/urfave/cli/v2"
+)
+
+var PlanCommand = cli.Command{
+	Name:  "plan",
+	Usage: "plan management",
+	Subcommands: cli.Commands{
+		&cli.Command{
+			Name:  "create",
+			Usage: "create a plan named `PLAN_NAME`",
+			Flags: []cli.Flag{
+				&cli.StringFlag{
+					Name:     "remote",
+					Usage:    "origin for your repository i.e. git@github.com:your/repo",
+					Required: false,
+				},
+				&cli.StringFlag{
+					Name:     "target",
+					Usage:    "target language (default: go), avail: [go]",
+					Required: false,
+					Value:    "go",
+				},
+				&cli.StringFlag{
+					Name:  "module",
+					Usage: "module name (used for initial templating",
+					Value: "github.com/your/module/name",
+				},
+				&cli.StringFlag{
+					Name:     "plan",
+					Aliases:  []string{"p"},
+					Usage:    "specifies the name of the plan to create",
+					Required: true,
+				},
+			},
+			Action: createCommand,
+		},
+		&cli.Command{
+			Name:  "import",
+			Usage: "import a plan from the local filesystem or git repository",
+			Flags: []cli.Flag{
+				&cli.StringFlag{
+					Name:     "source",
+					Usage:    "specifies the source of the plan to be imported, can be git or local.",
```

Comment:
I used a remote url as I would use with `git clone`, and I got this:

```
⟩ ./testground plan import --git --source git@github.com:libp2p/test-plans.git
Apr 30 13:25:39.726526	INFO	using home directory: /Users/raul/testground
Apr 30 13:25:39.726590	WARN	no .env.toml found at /Users/raul/testground/.env.toml; system may be misconfigured
parse "git@github.com:libp2p/test-plans.git": first path segment in URL cannot contain colon
```

Reply:
Would be great to document the format of the URLs this expects?

Reply:
I see why this is the case. I'm using url.Parse() to parse this string and as you point out, git sources are not URLs.

Reply:
Fixed now with a gitruls parser.